src/python/tests/core/bot/tasks/update_task_test.py

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 """update_task tests."""
```

Comment:
Probably revert these update_task_test, these need to fixed with deployment changes in another Cl by @oliverchang